giftcompare/categories/views.py

```diff
@@ -1,3 +1,16 @@
-from django.shortcuts import render
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from .models import Category
+from .serializers import CategorySerializer
+from rest_framework import status
 
-# Create your views here.
+class CategoryList(APIView):
+    def get(self, request):
+        categories = Category.objects.all()
+        serializer = CategorySerializer(categories, many=True)
+        return Response(serializer.data)
```

Comment:
please add a line between the def get and the def post to better readability

Reply:
Updated as requested